Keycloak/ci.yml

```diff
@@ -0,0 +1,12 @@
+prepare:
+  steps:
+    - name: Give Appropriate Permissions To The Install Script
+      command: chmod +x ./install.sh
+    - name: Run The Installation Script
+      command: ./install.sh
+test:
+  steps: []
+run:
+  steps:
+    - name: Start Keycloak
+      command: bash ./keycloak-$(cat ./KEYCLOAK_VERSION)/bin/kc.sh start --hostname $WORKSPACE_DEV_DOMAIN --http-port 3000 --proxy-headers xforwarded --http-enabled true
```

Comment:
Let's add the secrets as command line arguments from env vars here - see comment below for an example.